tests/framework/ROM/TimeSeries/ARMA/tests

```diff
@@ -74,4 +74,14 @@
     rel_err = 2e-1
     max_time = 300
   [../]
+
+  [./Segmented]
+    type = 'RavenFramework'
+    input = 'segmented.xml'
+    output = 'Segmented/synthetic.csv Segmented/synthetic_0.csv Segmented/synthetic.xml Segmented/stats.xml'
+    csv = 'Segmented/stats.csv'
+    rel_err = 1e-2
```

Comment:
can't we tight the tolerance more? If not, can you document why?

Reply:
See above. If you have any suggestions on improving the behavior between OS, I'd be happy to review a PR. We've discussed this many times in team meetings, and it is extraordinarily difficult to obtain identical behavior for more than three or four time steps when the operating system is changed.